tap_helpshift/__init__.py

```diff
@@ -145,7 +157,11 @@ def sync_stream_bg(self, stream_name, state, *args, start_date=None):
         if stream_name not in self.stream_counters:
             self.stream_counters[stream_name] = metrics.record_counter(stream_name)
         counter = self.stream_counters[stream_name]
-        instance = STREAMS[stream_name](self.client, start_date, sync_stream_bg=self.sync_stream_bg)
+        use_bookmarks = True
+        if self.is_secondary_syncer and (stream_name == 'issues' or stream_name == 'issue_analytics'):
```

Comment:
I think we can do this for all streams. The catalog on the secondary tap config can be used to determine which streams are synced this way.

Reply:
Yea, I was just guarding this to issues and issue analytics since those have been the ones that have the most discrepancies. I'll remove this conditional to allow all other streams on the catalog.